<launch>
    <node name="node_watcher" pkg="bot_sim" type="node_watcher_uc.py" output="screen" >
    </node>
</launch>